src/app.js

```diff
@@ -11,13 +12,20 @@ export default ({ DOM$ }) => {
 
   const cyclejs = Cyclejs({ MUSIC$: world.MUSIC$ })
 
+  const drivers = Drivers({ MUSIC$: cyclejs.MUSIC$ })
```

Comment:
I would put drivers component into cycleJS : 
 - Because drivers is a composition of cycleJS in real life
 - Because MUSIC$ stream is modified into cycleJS component by drivers component